apps/web/next.config.mjs

```diff
@@ -57,6 +57,7 @@ const config = {
     '@pancakeswap/hooks',
     '@pancakeswap/utils',
     '@pancakeswap/widgets-internal',
+    '@pancakeswap/universal-router-sdk'
```

Comment:
Let's publish our sdk before merge. Remember to remove the transpile after publish